test/integration/psetv2.spec.ts

```diff
@@ -366,15 +372,94 @@ describe('liquidjs-lib (transactions with psetv2)', () => {
       undefined,
       issuanceBlindingArgs,
     );
-    const blinder = new PsetBlinder(
+    let blinder = new PsetBlinder(
       pset,
       ownedInputs,
       zkpValidator,
       zkpGenerator,
     );
     await blinder.blindLast({ issuanceBlindingArgs, outputBlindingArgs });
-    const rawTx = signTransaction(pset, [alice.keys], Transaction.SIGHASH_ALL);
-    await regtestUtils.broadcast(rawTx.toHex());
+    const issuanceTx = signTransaction(
+      pset,
+      [alice.keys],
+      Transaction.SIGHASH_ALL,
+    );
+    const issuanceTxid = await regtestUtils.broadcast(issuanceTx.toHex());
+
+    const assetEntropy = issuanceEntropyFromInput(issuanceTx.ins[0]);
+    const reissuanceInputs = [
+      new CreatorInput(issuanceTxid, 0),
+      new CreatorInput(issuanceTxid, 3),
+    ];
+    const reissuanceOutputs = [
+      new CreatorOutput(
+        lbtc,
+        99998700,
+        alice.payment.output,
+        alice.payment.blindkey,
+        0,
+      ),
+      new CreatorOutput(lbtc, 700),
+    ];
+    const reissuancePset = PsetCreator.newPset({
+      inputs: reissuanceInputs,
+      outputs: reissuanceOutputs,
+    });
+
+    updater = new PsetUpdater(reissuancePset);
+    updater.addInWitnessUtxo(0, issuanceTx.outs[0]);
+    updater.addInWitnessUtxo(1, issuanceTx.outs[3]);
+    updater.addInUtxoRangeProof(0, issuanceTx.outs[0].rangeProof!);
+    updater.addInUtxoRangeProof(1, issuanceTx.outs[3].rangeProof!);
+    updater.addInReissuance(1, {
+      entropy: assetEntropy,
+      assetAmount: 1000,
+      assetAddress: alice.payment.confidentialAddress!,
+      tokenAmount: 1,
+      tokenAddress: alice.payment.confidentialAddress!,
+      tokenAssetBlinder: outputBlindingArgs[2].assetBlinder,
+    });
+    updater.addInSighashType(0, Transaction.SIGHASH_ALL);
+    updater.addInSighashType(1, Transaction.SIGHASH_ALL);
+
+    zkpGenerator = ZKPGenerator.fromInBlindingKeys([
+      alice.blindingKeys[0],
+      alice.blindingKeys[0],
+    ]);
+    const reissuanceownedInputs = await zkpGenerator.unblindInputs(
+      reissuancePset,
+    );
+    const reissuanceBlindingArgs = await zkpGenerator.blindIssuances(
+      reissuancePset,
+      {
+        1: alice.blindingKeys[0],
+      },
```

Comment:
Can we improve the API here to resemble more the `OwnedInput` data structure? just for sake of consistency in the blinding process? @altafan

Reply:
I don't think so. Blidning keys are required in the process to create asset/token rangeproofs. When blinding an issuance only the issued asset and token amount are blinded you know.